app/code/Magento/NewRelicReporting/Model/NewRelicWrapper.php

```diff
@@ -49,11 +49,24 @@ public function reportError($exception)
      */
     public function setAppName(string $appName)
     {
-        if (extension_loaded('newrelic')) {
+        if ($this->isExtensionInstalled()) {
             newrelic_set_appname($appName);
         }
     }
 
+    /**
+     * Wrapper for 'newrelic_name_transaction'
+     *
+     * @param string $transactionName
+     * @return void
+     */
+    public function setTransactionName(string $transactionName): void
```

Comment:
Adding a public function is a backward incompatible change, however, this class is not public (not marked with `@api` tag)